EnglishFilter/sections/specific.txt

```diff
@@ -270,6 +270,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+whatsnewinpublishing.com##.wp-block-spacer
```

Comment:
It doesn't contain ad - remove the rule..

Reply:
Ok.